src/addrman.h

```diff
@@ -348,8 +349,8 @@ friend class CAddrManTest;
      * Notice that vvTried, mapAddr and vVector are never encoded explicitly;
      * they are instead reconstructed from the other information.
      *
-     * vvNew is serialized, but only used if ADDRMAN_UNKNOWN_BUCKET_COUNT didn't change,
-     * otherwise it is reconstructed as well.
+     * vvNew is serialized, but only used if ADDRMAN_UNKNOWN_BUCKET_COUNT and the asmap checksum
```

Comment:
This comment is weird in general. I don't think we have anything like `ADDRMAN_UNKNOWN_BUCKET_COUNT` anymore?

Reply:
Should be talking about ADDRMAN_NEW_BUCKET_COUNT, ADDRMAN_NEW_BUCKETS_PER_ADDRESS.

Reply:
Thanks, changed to ADDRMAN_NEW_BUCKET_COUNT.